### DISCUSSION TITLE
Invalid Parameter -deskew

### DISCUSSION THREAD

**Discussion by dmcnaugh15** [ImageMagick/ImageMagick#7849] [Help] [answered]
### ImageMagick version

.7.1.1-41-Q16-HDRI-x64-dll

### Operating system, version and so on

Windows 11

### Description

I'm very new to ImageMagick.  Just installed it today, so bear with me.

When I try the deskew function, I get the following response: "Invalid Parameter - -deskew"

Here's the code:

```
convert Image.jpg -deskew 40% Image_deskew.jpg
```

Screenshot:
![Screenshot](https://github.com/user-attachments/assets/e018362d-4a10-4ada-875e-317463f13484)

**Comment by snibgo** [comment, marked as answer]
You are not running an ImageMagick program. Instead, you are running Microsoft "convert", which converts disk filing systems.

ImageMagick "convert" is deprecated. Instead, use "magick".

**Comment by dmcnaugh15** [comment]
Thank you @snibgo!
